### ISSUE TITLE
ag0 export --for-zero-height fails on mainnet

### ISSUE THREAD

**Issue by arirubinstein** [Agoric/Agoric#5682]
```
ari@mainnet:~/ag0$ time $(ag0 export --for-zero-height --height -1 > ~/genesis_export_optimized.json)
2:37AM INF asserting crisis invariants inv=0/11 module=x/crisis name=bank/nonnegative-outstanding
2:37AM INF asserting crisis invariants inv=1/11 module=x/crisis name=bank/total-supply
2:37AM INF asserting crisis invariants inv=2/11 module=x/crisis name=gov/module-account
2:37AM INF asserting crisis invariants inv=3/11 module=x/crisis name=distribution/nonnegative-outstanding
2:37AM INF asserting crisis invariants inv=4/11 module=x/crisis name=distribution/can-withdraw
2:39AM INF asserting crisis invariants inv=5/11 module=x/crisis name=distribution/reference-count
2:39AM INF asserting crisis invariants inv=6/11 module=x/crisis name=distribution/module-account
2:39AM INF asserting crisis invariants inv=7/11 module=x/crisis name=staking/module-accounts
2:39AM INF asserting crisis invariants inv=8/11 module=x/crisis name=staking/nonnegative-power
2:39AM INF asserting crisis invariants inv=9/11 module=x/crisis name=staking/positive-delegation
2:39AM INF asserting crisis invariants inv=10/11 module=x/crisis name=staking/delegator-shares
2:48AM INF asserted all invariants duration=628772.022129 height=5514676 module=x/crisis
panic: expected validator, not found

goroutine 1 [running]:
github.com/cosmos/gaia/v6/app.(*GaiaApp).prepForZeroHeightGenesis(0xc0011ad100, 0x2339fa0, 0xc0000400a8, 0x2353b20, 0xc000c63480, 0x0, 0x0, 0x0, 0x0, 0x5425b4, ...)
        github.com/cosmos/gaia/v6/app/export.go:163 +0x17b6
github.com/cosmos/gaia/v6/app.(*GaiaApp).ExportAppStateAndValidators(0xc0011ad100, 0xc001197b01, 0x307e078, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        github.com/cosmos/gaia/v6/app/export.go:29 +0x588
github.com/cosmos/gaia/v6/cmd/gaiad/cmd.appCreator.appExport(0x2343ec0, 0xc0010c1660, 0x2357120, 0xc0010fc400, 0x234ce60, 0xc00108aa00, 0xc0001f8ec0, 0x233ae20, 0xc001197bc0, 0x2352f20, ...)
        github.com/cosmos/gaia/v6/cmd/gaiad/cmd/root.go:294 +0x309
github.com/cosmos/cosmos-sdk/server.ExportCmd.func1(0xc00113aa00, 0xc0011a75f0, 0x0, 0x3, 0x0, 0x0)
        github.com/cosmos/cosmos-sdk@v0.44.5/server/export.go:71 +0x457
github.com/spf13/cobra.(*Command).execute(0xc00113aa00, 0xc0011a7590, 0x3, 0x3, 0xc00113aa00, 0xc0011a7590)
        github.com/spf13/cobra@v1.2.1/command.go:856 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000e09180, 0x0, 0x0, 0xc0010bf100)
        github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.2.1/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        github.com/spf13/cobra@v1.2.1/command.go:895
github.com/cosmos/cosmos-sdk/server/cmd.Execute(0xc000e09180, 0xc0010bf100, 0x11, 0x2357120, 0xc0010fc400)
        github.com/cosmos/cosmos-sdk@v0.44.5/server/cmd/execute.go:36 +0x285
main.main()
        github.com/cosmos/gaia/v6/cmd/ag0/main.go:52 +0x67

real    12m5.413s
user    18m17.450s
sys     1m7.456s
```

**Comment by Tartuffo** [comment]
Fixed in Gaia 7.

**Comment by arirubinstein** [comment]
https://github.com/cosmos/gaia/issues/1014

**Comment by Tartuffo** [comment]
Will be fixed by #5773.
